code/curve25519/Hacl.Curve25519_64_Slow.fsti

```diff
@@ -3,6 +3,24 @@ module Hacl.Curve25519_64_Slow
 open Hacl.Impl.Curve25519.Generic
 open Hacl.Impl.Curve25519.Fields
 
+[@@ Comment "Compute the scalar multiple of a point.
```

Comment:
I think we need to remove this file, as it's not supposed to be used. I can do it later

Reply:
Sounds good, thanks! I'll leave this part of the PR unchanged.